WordPress/Classes/ViewRelated/Media/SiteMedia/Controllers/SiteMediaCollectionViewController.swift

```diff
@@ -403,7 +403,7 @@ final class SiteMediaCollectionViewController: UIViewController, NSFetchedResult
         if let viewController = navigationController?.topViewController,
            let detailsViewController = viewController as? SiteMediaPageViewController {
             let before = indexPath.item > 0 ? fetchController.object(at: IndexPath(item: indexPath.item - 1, section: 0)) : nil
-            let after = indexPath.item < (fetchController.fetchedObjects?.count ?? 0) ? fetchController.object(at: IndexPath(item: indexPath.item + 1, section: 0)) : nil
+            let after = indexPath.item < (fetchController.fetchedObjects?.count ?? 0) ? fetchController.object(at: IndexPath(item: indexPath.item, section: 0)) : nil
```

Comment:
Just to verify, is this commit from Alex intended to be in this PR?

Reply:
Good catch. This has something to do with the previous base branch being `trunk`, and these changes were from the `23.7.1` hotfix. Commits from 23.7.1 have been merged to 23.8, and I've now updated the base branch to `release/23.8`.